frontend/src/App.js

```diff
@@ -53,8 +53,14 @@ function App() {
   const [playerUrl, setPlayerUrl] = useState(null)
 
   useEffect(() => {
-    fetch('/sessions')
-      .then(response => response.json())
+    fetch('/sessions', {redirect: "manual"})
+      .then(response => {
```

Comment:
you should check `response.ok` to make sure it was not an error, fetch will not auto throw like axios does